### ISSUE TITLE
Artifact for scala-native 0.4.3, scala 3 is empty

### ISSUE THREAD

**Issue by larsjaas** [scala/scala-parser-combinators#444]
Using scala-native 0.4.3, scala 3.1.1 and scala-parser-combinators 2.1.0 fails with import errors on scala.util.parsing.
Apparently the jar that was published for scala-native 0.4.3/scala 3 is empty (574 bytes).
https://search.maven.org/artifact/org.scala-lang.modules/scala-parser-combinators_native0.4_3/2.1.0/jar
Minimal demo repository: https://github.com/larsjaas/sq.git

**Comment by SethTisue** [comment]
@Philippus any idea who's attention we should call to this...?

**Comment by larsjaas** [comment]
The last commit (f2f0b4f89e1fc0ffb67bbc3f93567b866880a01b) updates the scala-native plugin to 0.4.3, and 0.4.3 is the first that has real Scala 3 support. Issue might already be fixed?

**Comment by ekrich** [comment]
It is using JUnit for tests too so that should be able to test on Native with the correct options. https://github.com/ekrich/sconfig/blob/main/build.sbt#L173-L176

**Comment by Philippus** [comment]
Going by the date, I assume the directories were created in https://github.com/scala/scala-parser-combinators/issues/418 for some reason.

**Comment by SethTisue** [comment]
@Philippus I don't understand what directories you're referring to?

> Issue might already be fixed in the source, just waiting for the next deployment?

Could someone try a `publishLocal` and see?

**Comment by Philippus** [comment]
The directories over here: https://repo1.maven.org/maven2/org/scala-lang/modules/scala-parser-combinators_native0.4_3/2.1.0/

**Comment by SethTisue** [comment]
I see. Well, I guess the history is only potentially worth digging into if someone confirms (via `publishLocal`) that the problem still exists.

**Comment by Philippus** [comment]
To me it looks ok:
```
ls -al docs/*.jar jars/*.jar
-rw-r--r--  1 philippus  staff  1554532  2 feb 20:53 docs/scala-parser-combinators_native0.4_2.13-javadoc.jar
-rw-r--r--  1 philippus  staff   511394  2 feb 20:53 jars/scala-parser-combinators_native0.4_2.13.jar
```

**Comment by larsjaas** [comment]
I figure it is maybe not time for a new release yet, but would it be possible to deploy a snapshot release to have something to use in scala-native? And get the fix (assuming #447 fixes this issue) verified.

**Comment by Philippus** [comment]
I've released 2.1.1.
@larsjaas I've tried your demo repo and that compiles and tests after updating scala-parser-combinators to 2.1.1.
